nri-postgresql/src/Postgres/Enum.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Postgres.Enum where
+
+import qualified Data.ByteString.Lazy as BSL
+import Data.List (group)
+import qualified Data.Text.Encoding as Encoding
+import qualified Language.Haskell.TH as TH
+import Database.PostgreSQL.Typed.Enum
+import Database.PostgreSQL.Typed.Protocol (pgSimpleQuery)
+import Database.PostgreSQL.Typed.TH (withTPGConnection)
+import Database.PostgreSQL.Typed.Dynamic (pgDecodeRep)
+import Set (Set)
+import qualified Set
+import qualified Prelude 
+
+
+quote :: Text -> Text
+quote t = "'" ++ t ++ "'"
```

Comment:
does this need to escape any `'` inside `t`?

Reply:
I thiiiink we can trust these names not to have a `'` in them

Reply:
doesn't feel safe. I know we don't use `'` in names a lot at NRI but it seems pretty common to prime-ify something to avoid a name collision in the rest of the ecosystem.

Reply:
Ohhh yeah I was thinking about prefixed `'` cause of all the TemplateHaskell, but I didn't think about postfix quotes